test/test_torch.py

```diff
@@ -98,6 +99,47 @@ def __exit__(self, *args):
         pass
 
 
+def _rebuild_subclass(type_, data, requires_grad, backward_hooks):
```

Comment:
I guess this code was cargo culted from somewhere? Might be worth saying where it came from.

Reply:
This was in torch/nn/parameter.py which was the original subclass of Tensor. The comment says it is redundant and it is. I can remove this is you want.

Reply:
I see the definition has now been moved to `torch._utils.py`

``` python
def _rebuild_parameter(data, requires_grad, backward_hooks):
    param = torch.nn.Parameter(data, requires_grad)
    # NB: This line exists only for backwards compatibility; the
    # general expectation is that backward_hooks is an empty
    # OrderedDict.  See Note [Don't serialize hooks]
    param._backward_hooks = backward_hooks

    return param
```